frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts

```diff
@@ -1,8 +1,8 @@
 import { useState } from 'react';
 
-type ReturnType = [boolean, () => void];
+type ToogleReturnType = [boolean, () => void];
```

Comment:
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md
check B17 rule and rename type
type UseToggleHook